Nos/Service/FeatureFlags.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+
+/// The set of feature flags used by the app.
+protocol FeatureFlags {
```

Comment:
While I like this style of architecture, and  given that you are using it in SwiftUI view, I wonder if we should make it a Dependency instead (just like I did with NamesAPI). The downside is that we continue to be depending on a 3rd party library, but the advantage is that you won't need to add` featureFlags: FeatureFlags = DefaultFeatureFlags()` to the constructor of every SwiftUI View that needs to check a flag.

Reply:
I agree -- I started adding unit tests and found that I'd like to switch this to `@Dependency`, so now that you suggested it I'll definitely do that.